internal/infrastructure/storage/db/pg/migration/20230504102232_init.up.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE market (
+    name VARCHAR(50) NOT NULL PRIMARY KEY,
```

Comment:
This implies that in other layers (domain, interface) we MUST validate the name specified by the user and check that length is <= 50.